mmv1/templates/terraform/examples/network_security_tls_inspection_policy_custom.tf.erb

```diff
@@ -0,0 +1,136 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_privateca_ca_pool" "default" {
+  provider = google-beta
+  name      = "<%= ctx[:vars]['privateca_ca_pool_name'] %>"
+  location  = "us-central1"
+  tier     = "DEVOPS"
+  publishing_options {
+    publish_ca_cert = false
+    publish_crl = false
+  }
+  issuance_policy {
+    maximum_lifetime = "1209600s"
+    baseline_values {
+      ca_options {
+        is_ca = false
+      }
+      key_usage {
+        base_key_usage {}
+        extended_key_usage {
+          server_auth = true
+        }
+      }
+    }
+  }
+}
+
+
+resource "google_privateca_certificate_authority" "default" {
+  provider = google-beta
+  pool = google_privateca_ca_pool.default.name
+  certificate_authority_id = "<%= ctx[:vars]['privateca_certificate_authority_id'] %>"
+  location = "us-central1"
+  lifetime = "86400s"
+  type = "SELF_SIGNED"
+  deletion_protection = false
+  skip_grace_period = true
+  ignore_active_certificates_on_deletion = true
+  config {
```

Comment:
we should leave one line between single line arguments and blocks

Reply:
done